microhapdb/marker.py

```diff
@@ -134,15 +134,22 @@ def parse_regionstr(regionstr):
     @staticmethod
     def standardize_ids(idents):
         def id_in_series(ident, series):
-            return series.str.contains(ident).any()
+            return ident in series.values
 
         ids = set()
         for ident in idents:
+            locusnames = microhapdb.markers.Name.apply(lambda x: x.split(".")[0])
```

Comment:
Initially, I supported microhap lookup by locus name with a pandas series `str.contains` operation. But that was a bit too permissive: that would allow `mh01` or `mh12KK` to be treated as identifiers and retrieve all IDs that begin with that prefix, which was *not* the intent of this update. So instead, I created a series of locus names (.v? stripped off) and now microhap ID queries must either match a full locus name or a full marker name to return a result.